spec/models/category_spec.rb

```diff
@@ -1,5 +1,35 @@
 require "rails_helper"
+require "support/factory_girl"
 
 RSpec.describe Category, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+
+  it "is invalid without a name" do
+    category = Category.create()
+    expect(category).to_not be_valid
+  end
+
+  it "requires a name to be valid" do
+    category = Category.create(name: "Specials")
+    expect(category).to be_valid
+  end
+
+  it 'is valid when created with factory_girl' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.